homeassistant/components/recorder/migration.py

```diff
@@ -498,10 +520,18 @@ def _create_index(
             connection = session.connection()
             index.create(connection)
         except (InternalError, OperationalError, ProgrammingError) as err:
+            if _unindexable_legacy_column(instance, base, err):
+                _LOGGER.info(
```

Comment:
```suggestion
                _LOGGER.debug(
```

There is nothing we can do and nothing is actually wrong so maybe debug instead?